server/network/aoprotocol.py

```diff
@@ -813,28 +833,29 @@ def net_cmd_casea(self, args):
                 return
 
             if not args[1] == "1" and not args[2] == "1" and not args[
-                    3] == "1" and not args[4] == "1" and not args[5] == "1":
+                    3] == "1" and not args[4] == "1" and not args[5] == "1" and not args[6] == "1":
```

Comment:
Will cause KeyError when sent by 2.6 clients.

Reply:
Fixed by changing it to `if not ("1" in args[1:]):`